samples/nrf9160/nrf_cloud_mqtt_multi_service/prj.conf

```diff
@@ -16,6 +16,15 @@ CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_RESET_ON_FATAL_ERROR=y
 CONFIG_NCS_SAMPLES_DEFAULTS=y
 
+# Improved Logging
+# CONFIG_LOG_MODE_DEFERRED allows logging from multiple threads simultaneously without creating
+# splintered log entries, at the cost of needing a buffer for storing logs
```

Comment:
The next entry has a period at the end of the sentence, but this entry and the ones following do not.  You might want to be consistent.